config/filemapprovider/mapprovider_test.go

```diff
@@ -26,42 +26,42 @@ import (
 	"go.opentelemetry.io/collector/config"
 )
 
-const fileSchemePrefix = fileSchemeName + ":"
+const fileSchemePrefix = schemeName + ":"
 
-func TestFile_EmptyName(t *testing.T) {
-	fp := NewFile()
+func TestEmptyName(t *testing.T) {
+	fp := New()
 	_, err := fp.Retrieve(context.Background(), "", nil)
 	require.Error(t, err)
 	require.NoError(t, fp.Shutdown(context.Background()))
 }
 
-func TestFile_UnsupportedScheme(t *testing.T) {
-	fp := NewFile()
-	_, err := fp.Retrieve(context.Background(), "http://", nil)
+func TestUnsupportedScheme(t *testing.T) {
+	fp := New()
+	_, err := fp.Retrieve(context.Background(), "https://", nil)
```

Comment:
Is this intended? It was http before, it's https now

Reply:
Will split in a separate PR. My IDE complains `http links are not secure` :))